.gitignore

```diff
@@ -18,3 +18,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode/extensions.json
```

Comment:
Please remove this line. We _want_ to track this so that we can share extension recommendations.